var/spack/repos/builtin/packages/eic-opticks/package.py

```diff
@@ -0,0 +1,40 @@
+from spack.package import *
+
+
+class EicOpticks(CMakePackage, CudaPackage):
+    """EIC Opticks package for GPU simulation"""
+
+    homepage = "https://github.com/bnlnpps/eic-opticks"
+    url      = "https://github.com/bnlnpps/eic-opticks/archive/tags/1.0.0-rc1.tar.gz"
+    git      = "https://github.com/bnlnpps/eic-opticks.git"
+
+    maintainers = ["plexoos"]
```

Comment:
Use directive syntax.
```suggestion
    maintainers("plexoos")
```